### PR TITLE
Update Head.php

### PR DESCRIPTION
**Pull request by priyanka-saini**
Issue:
1. There were some JS error after installing the module.
2. When the Js and Css files are not merged then timestamp was added only to those files which are in skin folder (No timestamp was there for Js and Css files which are in Js or some other folder).

### CODE REVIEW

File: app/code/community/Aoe/JsCssTstamp/Block/Head.php [hunk 1]
```diff
@@ -264,39 +264,56 @@ protected function _separateOtherHtmlHeadElements(&$lines, $itemIf, $itemType, $
      */
     protected function &_prepareStaticAndSkinElements($format, array $staticItems, array $skinItems, $mergeCallback = null)
     {
+        $versionKey = Mage::getModel('aoejscsststamp/package')->getVersionKey();
```
Comment (norgeindian) [inline app/code/community/Aoe/JsCssTstamp/Block/Head.php]: This function is protected. To make it work, it should be public. 
Please implement this in your pull request.

File: app/code/community/Aoe/JsCssTstamp/Block/Head.php [hunk 2]
```diff
@@ -264,39 +264,56 @@ protected function _separateOtherHtmlHeadElements(&$lines, $itemIf, $itemType, $
...
+        $AddTstampToAssetsJs = Mage::getModel('aoejscsststamp/package')->getAddTstampToAssetsJs();
```
Comment (norgeindian) [inline app/code/community/Aoe/JsCssTstamp/Block/Head.php:268]: This call does not work. I would recommend to change it to:
$AddTstampToAssetsJs = Mage::getStoreConfig('dev/js/addTstampToJsFiles');

File: app/code/community/Aoe/JsCssTstamp/Block/Head.php (hunk 1 above)
Comment (norgeindian) [inline app/code/community/Aoe/JsCssTstamp/Block/Head.php:267]: This function is protected. To make it work, it should be public.
Please implement this in your pull request.

**Review by norgeindian — CHANGES_REQUESTED** [review]

**Comment by sprankhub** [comment]
Fixed with #12.

**Pull request closed without merging**